src/components/NewTransaction/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useTransactionContext } from '../../context/globalState';
+import { StyledNewTransaction } from './styled';
+import Button from '../Button';
+import RadioBtn from '../RadioBtn';
+
+const NewTransaction = ({item, setReadOnly}) => {
```

Comment:
I'd rename the component (and the folder) to `TransactionForm`, since it's not only about adding new Transaction only now